types/aframe/tests/package.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
I might be able to run the tests w/ `tests.html` and load the test framework via unpkg.org so `package.json` would not be necessary. The tests are important to help identify any changes in aframe as new versions are released. Does DT have any standards for providing tests that I could follow?